Api/Controllers/FileController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Application.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+[ApiController]
+[Route("api/[controller]")]
+public class FilesController : ControllerBase
```

Comment:
Protect this controller with `[Authorize]` attribute and create TODO command to make this controller available only for admins